scripts/build_dev_server_docker

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o xtrace
+
+readonly CACHE_DIR='bazel-cache'
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+"$SCRIPTS_DIR/docker_run" bazel \
+    --output_base="$CACHE_DIR/dev" \
+    build \
```

Comment:
The args and flags seem in a weird order -- is it possible to keep `bazel build` together?

Reply:
Nope. Apparently output_base is an arg for bazel, not for build :(